apple/swift-docc
# Fix wrong role assigned to technology-root articles

Bug/issue #, if applicable: #743
(rdar://117019142 rdar://116419389)

## Summary

This PR includes a fix for an error that was causing technology root articles to have the wrong role assigned in their render node.

## Dependencies

N/A

## Testing

[Article.docc.zip](https://github.com/apple/swift-docc/files/13043268/Article.docc.zip)

Steps:
1. Download the attached catalog.
2. Run `docc convert Article.docc -o Article.doccarchive`.
3. Within the generated .doccarchive, navigate to data-> documentation-> article.json
4. Open article.json
5. In the JSON file, verify the value associated with the key `role` is `collection`

For additional testing, you can perform the same steps for a multiple-article-only catalog where the technology root doesn't have a `Topics` section.

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.
